templates/hotgraphicPopup.jsx

```diff
@@ -50,7 +50,7 @@ export default function HotgraphicPopup(props) {
 
               {title &&
               <div
-                id={_isActive && 'notify-heading'}
+                id={_isActive ? 'notify-heading': null}
```

Comment:
There should be a space after `:`.

Reply:
Sorry, would normally have done this, so just a typo. Will amend accordingly.